hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/event/SoftwareModuleEvent.java

```diff
@@ -41,6 +43,16 @@ public SoftwareModuleEvent(final BaseEntityEventType entityEventType, final Soft
     }
 
     /**
+     * Delete software modules
```

Comment:
It is the same in the other events. That's a default construtor which set the event type to deleted.